include/tvm/tir/analysis.h

```diff
@@ -30,6 +30,7 @@
 #include <tvm/tir/function.h>
 #include <tvm/tir/op_attr_types.h>
 #include <tvm/tir/stmt.h>
+#include <tvm/tir/stmt_functor.h>
```

Comment:
Is this include file required, now that the visitor/mutator are in a separate header file?